here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresStorage.java

```diff
@@ -450,6 +439,50 @@ synchronized void initStorage(@NotNull PsqlStorage.Params params, @NotNull IoHel
     }
   }
 
+  private void initStoragePlv8(
+      PsqlStorage.@NotNull Params params, @NotNull IoHelp ioHelp, Statement stmt, PsqlConnection conn)
```

Comment:
From supplied arguments only the `conn` one is used - I guess we'll be fine removing other ones (unless there is some reflection-related magic that enforces specific arg set on us)

Reply:
Fixed.